src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -283,7 +283,7 @@ private bool PostProcessTryGetValue(CoherentState coherentState, CacheEntry? ent
 
                     StartScanForExpiredItemsIfNeeded(utcNow);
                     // Hit
-                    if (_allStats is not null)
+                    if (_stats is not null)
```

Comment:
Does this have any effect?

Reply:
No, but I changed it for clarity - `GetStats()` uses `_stats!.Value`, so this check now matches the null-suppression.